docs/user/dql/functions.rst

```diff
@@ -1711,6 +1711,81 @@ Example::
     +----------------------------+----------------------------+
 
 
+
+UTC_DATE
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC date as a value in 'yyyy-MM-dd'.
+
+Return type: DATE
+
+Specification: UTC_DATE() -> DATE
+
+Example::
+
+    >od SELECT UTC_DATE();
+    fetched rows / total rows = 1/1
+    +--------------+
+    | utc_date()   |
+    |--------------|
+    | 2022-10-03   |
+    +--------------+
+
+
+
+UTC_TIME
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC time as a value in 'HH:mm:ss'.
+
+Return type: TIME
+
+Specification: UTC_TIME() -> TIME
+
+Example::
+
+    >od SELECT UTC_TIME();
+    fetched rows / total rows = 1/1
+    +--------------+
+    | utc_time()   |
+    |--------------|
+    | 17:54:27     |
+    +--------------+
+
+
+
+
+UTC_TIMESTAMP
+-------
+
+Description
+>>>>>>>>>>>
+
+Returns the current UTC timestamp as a value in 'yyyy-MM-dd HH:mm:ss'.
+
+Return type: DATETIME
+
+Specification: UTC_TIMESTAMP() -> DATETIME
```

Comment:
Feels like this should be returning a `TIMESTAMP`?

Reply:
Timestamp is formatted as a datetime. We have no `exprTimeStamp`.